internal/gossiper/manual.go

```diff
@@ -137,8 +137,6 @@ func (g *Manual[T]) HandleAppGossip(ctx context.Context, nodeID ids.NodeID, msg
 	return nil
 }
 
-func (*Manual[T]) BlockVerified(int64) {}
```

Comment:
The no-ops `BlockVerified` and `Queue` should not need to be part of a shared interface between `Manual` and `Proposer` if they register their own notification requirements from the VM rather than rely on the VM to call `BlockVerified` or `Queue` directly